.eslintrc.js

```diff
@@ -152,6 +152,7 @@ module.exports = {
             files: ["web/src/**"],
             globals: {
                 StripeCheckout: false,
+                Gtag: false,
```

Comment:
This is required so that we can use `Gtag` globally without `eslint` complaining.

**NOTE:** `false` tells `eslint` that this is an immutable global.

Reply:
It’s clearer (and avoids invalidating the ESLint cache) to put a `/* global Gtag */` [comment](https://eslint.org/docs/latest/use/configure/language-options#specifying-globals) in the one file using `Gtag`.